students/Wayne/Assignments/Lesson6/test_Mailroom.py

```diff
@@ -0,0 +1,16 @@
+
+
+"""
+
+Built to test the Mailroom script for bugs
+
+"""
+
+
+import Mailroom2 as MR
+
+
+def test_accept_donation():
+    amount = 1400
+    MR.accept_donation( amount)
+    assert MR.donor_db == {[1400]}
```

Comment:
This test seems misleading. Your accept_donation() expects a name, not an amount. Tests should reflect the spec of the code that's tested.